### PR TITLE
add r-packrat

### PR DESCRIPTION
**Pull request by daler**
(just one recipe for now while I make sure I understand the build system)

xref #2009

### CODE REVIEW

**Comment by daler** [comment]
@conda-forge/core it's unclear from the docs whether the merges happen automatically, if we have to apply for group membership, or core members do the merge. Anything to do on my end here?

File: recipes/r-packrat/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (johanneskoester) [inline recipes/r-packrat/build.sh]: Above two lines can be removed.

Comment (johanneskoester) [inline recipes/r-packrat/build.sh]: Above two lines can be removed. They are only needed for the core R packages.

File: recipes/r-packrat/bld.bat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
```
Comment (johanneskoester) [inline recipes/r-packrat/bld.bat]: Can you remove the comments below?

**Comment by johanneskoester** [comment]
When a PR builds, somebody of conda-forge/core has to merge it.

File: recipes/r-packrat/build.sh (hunk 1 above)
Comment (daler) [inline recipes/r-packrat/build.sh]: Thanks, I didn't realize it was package-specific.

**Comment by daler** [comment]
redundant with #2812; closing.

**Comment by jakirkham** [comment]
It would be great if someone from bioconda could come to the conda-forge meeting and discuss R packages. We are deciding on the date now. Please feel free to add in your availability in the linked poll if you are interested in coming.

https://groups.google.com/forum/#!topic/conda-forge/1zLBSOPRDhk

**Comment by johanneskoester** [comment]
I would like to join. I have added me into the poll.

**Comment by jakirkham** [comment]
Awesome, thanks @johanneskoester. Feel free to share this info with others.

**Comment by johanneskoester** [comment]
@bgruening @daler @croth1 @jrderuiter: if you are interested in joining a conda-forge meeting where R migration will be discussed, see above doodle for finding a date.

**Comment by croth1** [comment]
@johanneskoester, thanks for the invitation. Unfortunately, I'll be very busy the next weeks. Can you please put the cran timeouts on the list of things to discuss? I think we need a viable solution there otherwise the R packaging will end up wasting a lot of ci time.

**Comment by johanneskoester** [comment]
Very good point. I have added it to the hackpad.

**Pull request closed without merging**